extended-descriptions/explainer-extended-descriptions.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Explainer: Extended Descriptions for Publications and the Web"
+date: 11-27-2025
+author: Publishing Maintenance Group Accessibility Task Force
+---
+
+# Explainer: Extended Descriptions for Publications and the Web
+
+## Authors
+
+- Publishing Maintenance Group Accessibility Task Force
+
+## Participate
+
+- Issue tracker: 
+- Discussion: 
+
+## Introduction
+
+This explainer documents the need for and proposed semantics for extended descriptions: longer, structured content that explains images, diagrams, tables, formulas, and other information-rich material that cannot be adequately represented by brief alternative text alone but needs to be closely linked to the image it explains.
+
+The document summarizes goals, non-goals, candidate approaches, examples, alternatives considered, and next steps to enable review by APA, TAG, implementers and the community.
+
+## User-Facing Problem
+
+Users encounter images and non-text content that convey complex information (e.g., technical diagrams, charts, mathematical notation, museum objects) where a short `alt` is insufficient and an extended, structured description is provided. Without clear affordances, users may not discover these descriptions or understand the linking between them and the primary content.
+
+Authors and reading systems need a reliable, discoverable, and programmatic way to identify and surface extended descriptions in consistent ways without breaking reading flow or excluding non-AT users.
```

Comment:
Mixing authors and reading systems here is kind of confusing. I got to the part about surfacing descriptions and thought maybe AT was intended instead of author. I'd suggest separating the authoring and device concerns and adding reading systems and AT together for the need to programmatically identify and report semantics. Otherwise, it leaves open the question of what benefit this has for AT and that could lead to pushback that this belongs in epub:type.